### ISSUE TITLE
Games in 

### ISSUE THREAD

**Issue by hlabrand** [iftechfoundation/ifdb#45]

**Comment by dfabulich** [comment]
I reproduce this! Thanks for reporting it.

**Comment by lkcampbell** [comment]
Yup, it looks like all of the page navigation links (i.e. Next, Previous and any of the numbers) are broken. I will take a look at this issue.

**Comment by lkcampbell** [comment]
Okay, got my commits and pull requests working correctly now. Pull request #47 is the cleaned up version of pull request #46. Thanks.
